### ISSUE TITLE
DreamBooth and DeepSpeed

### ISSUE THREAD

**Issue by Dullson** [TheLastBen/fast-stable-diffusion#100]
There was a [reddit thread](https://www.reddit.com/r/StableDiffusion/comments/xwdj79/dreambooth_training_in_under_8_gb_vram_and/) claiming to run dreambooth under 8GB VRAM using [DeepSpeed](https://www.deepspeed.ai/), they also shared [their code](https://github.com/Ttl/diffusers/tree/dreambooth_deepspeed). I don't see anyone in the replies reporting success and I also failed to make it work. Maybe you or someone in the community can figure it out and we can lower VRAM requirements further.
[Here](https://gist.github.com/Dullson/51605b6e291e551622a34515e6a246c7) is my test notebook for paperspaces if you want to check it out.

**Comment by TheLastBen** [comment]
I saw that, it requires at least 25GB of RAM which is inconvenient for Google Colab since it offers 16GB GPUs and 12GB of RAM

**Comment by Dullson** [comment]
Yes, it's not suitable for colab but paperspace offers 8GB VRAM and 30GB RAM on their lowest free tier machines. Almost like a perfect candidate for this. Their running times are not as restricted. This might be a good way to relieve some pressure from colab.

**Comment by 78Alpha** [comment]
I have tried this out myself, 8 GB GPU and 48 GB RAM, and most others that have tried it have had no success. It will always OOM. Even on a 16 GB card  with 64 GB RAM people are running into OOM.
